.github/workflows/first_startup.yaml

```diff
@@ -53,6 +55,10 @@ jobs:
         with:
           path: .tox
           key: tox-cache-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('galaxy root/requirements.txt') }}-first-startup
+      - uses: actions/cache@v3
+        with:
+          key: galaxy-static-${{ needs.build-client.outputs.commit-id }}
+          path: 'galaxy root/static'
```

Comment:
Can this block just be put at the end of the build_client.yaml instead of duplicated?

Reply:
My understanding is that there are no shared disks between jobs?

Reply:
No, you are right, I was misremembering how the cache action work, this is brilliant!
Maybe can you add a `name: ` attribute though?